### DISCUSSION TITLE
Embed a movie in presentation

### DISCUSSION THREAD

**Discussion by schoeps** [tudace/tuda_latex_templates#397] [General LaTeX Questions]
What are the best options ton include movies in presentations? Is there anything beyond https://www.overleaf.com/learn/latex/Questions/How_can_I_embed_a_video_in_my_PDF_using_LaTeX? I currently use `multimedia.sty` with `\movie` but this has issues on Windows and does not really embed the movie (you still need to distribute the movie separately). Anything better available?

**Comment by TeXhackse** [comment]
Actually there is no really good option on this. Especially since flash is no longer available. Embedding the video either requires a specific player plugin for the viewer which is restricting it to a specific setup or also embedding the player into the pdf file. 

So it really depends on if you know the setup where you want to play the files on. To remain portability I'd recommend linking the videos instead of embedding them. It's possible to add the movie as a file attachment but this is confusing most viewers. I cannot recommend embedding videos when you don't know which viewer on which OS is going to be used.
